### DISCUSSION TITLE
multi- vehicle using gazebo drone project in airsim

### DISCUSSION THREAD

**Discussion by airsimdevdd** [microsoft/AirSim#4220] [Support Q&A]
Is multi- vehicle possible in airsim using gazebodrone project

**Comment by zimmy87** [comment]
Currently GazeboDrone just subscribes to topics for 1 drone in Gazebo and updates the pose for the default drone in AirSim. GazeboDrone would need to be updated to use vehicle names in order to enable a multi-vehicle scenario.

**Reply by airsimdevdd** [reply]
can any one help me with  this

**Reply by HarrySoteriou** [reply]
I am also interested in this feature, but I am still in the process of integrating a single drone model from gazebo to AirSim. Will have a look at it but I only know Python so it will definitely take me a couple of months
